build_packages

```diff
@@ -285,6 +286,48 @@ export KBUILD_BUILD_HOST="${BUILD_HOST:-pony-truck.infra.kinvolk.io}"
 info "Merging board packages now"
 sudo -E "${EMERGE_CMD[@]}" "${EMERGE_FLAGS[@]}" "$@"
 
+info "Merging sysext packages now"
+for sysext in "${EXTRA_SYSEXTS[@]}"; do
+  IFS="|" read SYSEXT_NAME PACKAGE_ATOMS USEFLAGS ARCHES < <(echo "$sysext");
+
+  arch_array=(${ARCHES//,/ })
+  if [[ ! -z "$ARCHES" ]]; then
+    should_skip=1
+    for arch in "${arch_array[@]}"; do
+      if [ "$arch" == "$ARCH" ]; then
+        should_skip=0
+      fi
+    done
+    if [ "$should_skip" -eq 1 ]; then
+      continue
+    fi
+  fi
+
+
+  info "Building packages for $SYSEXT_NAME sysext with USE=$USEFLAGS"
+  IFS=,
+  for package in $PACKAGE_ATOMS; do
+     # --buildpkgonly does not install dependencies, so we install them
+     # separately before building the binary package
+     sudo --preserve-env=MODULES_SIGN_KEY,MODULES_SIGN_CERT \
+       env USE="$USEFLAGS" FEATURES="-ebuild-locks binpkg-multi-instance" "${EMERGE_CMD[@]}" \
+       ${EMERGE_FLAGS[@]} \
```

Comment:
You need to quote this to use it as an array and avoid unintended word splitting.

```suggestion
       "${EMERGE_FLAGS[@]}" \
```